torch/csrc/jit/tensorexpr/ir_simplifier.h

```diff
@@ -372,7 +372,9 @@ class MinTerm : public ExprNode<MinTerm> {
 };
 
 // Context-sensitive IR simplification
-using VarBoundInfo = std::unordered_map<VarPtr, std::pair<ExprPtr, ExprPtr>>;
+using LoopBoundInfo = std::pair<ExprPtr, ExprPtr>;
```

Comment:
It seems that this definition is not necessary. See the comments above.

Reply:
I removed the `LoopBoundInfo` and use `Bound`